itest/opstackl2/op_test_manager.go

```diff
@@ -394,24 +451,6 @@ func startEotsManagers(
 	return eh, eotsClients
 }
 
-func initSdkClient(
-	opSys *ope2e.System,
-	opL2ConsumerConfig *fpcfg.OPStackL2Config,
-	t *testing.T,
-) *sdkclient.SdkClient {
-	// We pass in an external Bitcoin RPC address but otherwise use the default configs.
-	btcConfig := btcclient.DefaultBTCConfig()
-	// The RPC url must be trimmed to remove the http:// or https:// prefix.
-	btcConfig.RPCHost = trimLeadingHttp(opSys.Cfg.DeployConfig.BabylonFinalityGadgetBitcoinRpc)
```

Comment:
is `trimLeadingHttp` still being used after removing this function? if not let's remove it